generators/Exercises/Alphametics.cs

```diff
@@ -24,7 +25,6 @@ public Alphametics()
         protected override TestClass CreateTestClass()
         {
             var testClass = base.CreateTestClass();
-            testClass.UsingNamespaces.Add(typeof(ArgumentException).Namespace);
             testClass.UsingNamespaces.Add(typeof(Dictionary<char, int>).Namespace);
```

Comment:
I had this at some point, but for some reason decided against it. Will add it back!